### PR TITLE
Corrected Issue with normalized time index and updated testing

### LINKED ISSUE #750: Normalize entity bug: putting base entity time index in additional_variables
The current logic in `normalize_entity` allows making the base entity's time index an additional variable.  This results in the time index column being removed from the base entity.  The `time_index` attribute of the base entity is still set to that column, however.

Code example:
```python
def test_normalize_time_index_as_additional_variable(es):
    assert "signup_date" in es["customers"].df.columns
    es.normalize_entity(base_entity_id='customers',
                        new_entity_id='cancellations',
                        index='cancel_reason',
                        make_time_index='signup_date',
                        additional_variables=['signup_date'],
                        copy_variables=[])
    assert "signup_date" in es["customers"].df.columns
```

### PR DESCRIPTION
**Pull request by ablacke-ayx**
### Pull Request Description
Fixes #750 - Stop user from removing time index from base_entity using the additional_variables 

-----
*After creating the pull request: in order to pass the **changelog_updated** check you will need to update the "Future Release" section of* `docs/source/changelog.rst` *to include this pull request.*

### CODE REVIEW

File: featuretools/tests/entityset_tests/test_es.py [hunk 1]
```diff
@@ -15,6 +15,19 @@
...
+    error_text = "Not moving signup_date as it is the base time index variable."
+    with pytest.raises(ValueError, match=error_text):
+        assert "signup_date" in es["customers"].df.columns
+        es.normalize_entity(base_entity_id='customers',
+                            new_entity_id='cancellations',
+                            index='cancel_reason',
+                            make_time_index='signup_date',
+                            additional_variables=['signup_date'],
+                            copy_variables=[])
+        assert "signup_date" in es["customers"].df.columns
```
Comment (rwedge) [inline featuretools/tests/entityset_tests/test_es.py]: This line will not get hit due to the previous one raising an error

File: featuretools/entityset/entityset.py [hunk 2]
```diff
@@ -566,12 +566,16 @@ def normalize_entity(self, base_entity_id, new_entity_id, index,
         for v in additional_variables + copy_variables:
             if v == index:
                 raise ValueError("Not copying {} as both index and variable".format(v))
-                break
+
+        for v in additional_variables:
+            if v == base_entity.time_index:
+                raise ValueError("Not moving {} as it is the base time index variable.".format(v))
```
Comment (rwedge) [inline featuretools/entityset/entityset.py]: Let's suggest ways the user can resolve the error, either by setting `make_time_index` to `True` so it copies the base entity's time index by default, or by including the variable in `copy_variables`

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -13,7 +14,7 @@ Changelog
     * Testing Changes
 
     Thanks to the following people for contributing to this release:
-    :user:`kmax12`, :user:`rwedge`
+    :user:`kmax12`, :user:`rwedge`,:user:`ablacke-ayx`
```
Comment (rwedge) [inline docs/source/changelog.rst]: add a space between the comma and your user tag

File: featuretools/tests/entityset_tests/test_es.py (hunk 1 above)
Comment (ablacke-ayx) [reply featuretools/tests/entityset_tests/test_es.py]: ??

**Comment by ablacke-ayx** [comment]
Yes. I will on Monday.

Sent from my T-Mobile 4G LTE Device
Get Outlook for Android<https://aka.ms/ghei36>

________________________________

File: featuretools/tests/entityset_tests/test_es.py (hunk 1 above)
Comment (rwedge) [reply featuretools/tests/entityset_tests/test_es.py]: The `normalize_entity` call will raise an error, which stops code execution within the with statement.  If we wanted the final assert statement to be run after the error to confirm we didn't remove the signup date column, we would need to move the assert out of the `with pytest.raises` block

File: docs/source/frequently_asked_questions.ipynb [hunk 4]
```diff
@@ -148,19 +146,19 @@
...
    "source": [
     "Above, we normalized the columns to create a new entity. \n",
-    "- For `additional_variables`, the following columns `['session_start', 'join_date]` will be removed from the `products` entity, and moved to the new `device` entity. \n",
+    "- For `additional_variables`, the following column `['join_date]` will be removed from the `products` entity, and moved to the new `device` entity. \n",
     "\n",
     "\n",
-    "- For `copy_variables`, the following columns `['device', 'customer_id', 'zip_code']` will be copied from the `products` entity to the new `device` entity. \n",
+    "- For `copy_variables`, the following columns `['device', 'customer_id', 'zip_code','session_start']` will be copied from the `products` entity to the new `device` entity. \n",
     "\n",
     "Let's see this in the actual `EntitySet`."
```
[the remark this answers is not in the document]
Comment (ablacke-ayx) [reply docs/source/frequently_asked_questions.ipynb]: Updated

File: docs/source/frequently_asked_questions.ipynb [hunk 5]
```diff
@@ -238,7 +234,7 @@
     "                         new_entity_id=\"sessions\",\n",
     "                         index=\"session_id\",\n",
     "                         make_time_index=\"session_start\",\n",
-    "                         additional_variables=[\"session_start\"])\n",
```
[the remark this answers is not in the document]
Comment (ablacke-ayx) [reply docs/source/frequently_asked_questions.ipynb]: You are right.

File: featuretools/tests/entityset_tests/test_es.py [hunk 6]
```diff
@@ -694,12 +706,6 @@ def test_normalize_entity_new_time_index_error_check(es):
                             index='cancel_reason',
                             make_time_index='cancel_date')
 
-    es.normalize_entity(base_entity_id='customers',
```
Comment (rwedge) [inline featuretools/tests/entityset_tests/test_es.py]: Can we add this back in to test the "not the original time index but still using additional variables" case

**Pull request merged**